apps/frontend/src/components/insightConfiguration/InsightConfiguration.tsx

```diff
@@ -89,9 +89,8 @@ export const InsightConfiguration = ({ insight, setInsight, errorMessages }: IPr
                         <BodyShort>til</BodyShort>
                         <Datepicker value={insight.end} onChange={(value) => handleDateChange(value, 'end')} />
                     </div>
-                    {errorMessages.datesErrorMsg!.length > 0 && (
-                        <ErrorList errorMessages={errorMessages.datesErrorMsg!}/>
-                    )}
+                    {validationErrors.start && <ErrorList errorMessages={[...validationErrors.start]} />}
+                    {validationErrors.end && <ErrorList errorMessages={[...validationErrors.end]} />}
```

Comment:
Using this will be less consistent with the error messages coming from other input fields. I believe it would be confusing for the user if we were to use both.